<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ravi" name="test" tests="2" failures="1" timestamp="25 Feb 2018 05:01:26 GMT" time="55.081" errors="0">
  <testcase name="feature" time="13.509" classname="com.testframe.runner.Login.Loginrunner"/>
  <testcase name="feature" time="40.865" classname="com.testframe.runner.NewCustomer.newcustomerrunner">
    <failure type="cucumber.runtime.CucumberException" message="cucumber.runtime.CucumberException: Failed to instantiate class com.testframe.stepdefinition.NewCustomer.AddNewCustomer">
      <![CDATA[cucumber.runtime.CucumberException: cucumber.runtime.CucumberException: Failed to instantiate class com.testframe.stepdefinition.NewCustomer.AddNewCustomer
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
Caused by: cucumber.runtime.CucumberException: Failed to instantiate class com.testframe.stepdefinition.NewCustomer.AddNewCustomer
	at cucumber.runtime.java.DefaultJavaObjectFactory.cacheNewInstance(DefaultJavaObjectFactory.java:46)
	at cucumber.runtime.java.DefaultJavaObjectFactory.getInstance(DefaultJavaObjectFactory.java:32)
	at cucumber.runtime.java.JavaStepDefinition.execute(JavaStepDefinition.java:38)
	at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)
	at cucumber.runtime.Runtime.runStep(Runtime.java:299)
	at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)
	at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)
	at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:44)
	at cucumber.runtime.model.CucumberScenarioOutline.run(CucumberScenarioOutline.java:46)
	at cucumber.runtime.model.CucumberFeature.run(CucumberFeature.java:165)
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:63)
	... 25 more
	at cucumber.runtime.java.DefaultJavaObjectFactory.cacheNewInstance(DefaultJavaObjectFactory.java:40)
	... 35 more
Caused by: java.lang.NullPointerException
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy18.isDisplayed(Unknown Source)
	at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:302)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$100(ExpectedConditions.java:44)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:288)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:285)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:209)
	at framework.helper.Wait.WaitHelper.waitForElement(WaitHelper.java:56)
	at framework.PageObject.NewCustomerpage.<init>(NewCustomerpage.java:56)
	at com.testframe.stepdefinition.NewCustomer.AddNewCustomer.<init>(AddNewCustomer.java:9)
	... 40 more
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- test -->
